Celeste.Mod.mm/Patches/Autotiler.cs

```diff
@@ -241,11 +241,16 @@ public patch_Autotiler(string filename)
             char tile = GetTile(mapData, x, y, forceFill, forceID, behaviour);
             if (IsEmpty(tile))
                 return null;
-            patch_TerrainType terrainType;
-            try { // Satisfies error handling for the orig_ method too.
-                terrainType = lookup[tile];
-            } catch (KeyNotFoundException e) {
-                throw new KeyNotFoundException($"Level contains a tileset with an id of '{tile}' that is not defined.", e);
+
+            // Satisfies error handling for the orig_ method too.
+            if (!lookup.TryGetValue(tile, out patch_TerrainType terrainType)) {
+                throw new KeyNotFoundException { 
```

Comment:
The exception still eventually gets logged, so I think it should probably also have a human-readable error message as before.